### ISSUE TITLE
Error in test suite when running under Python 3.4.4

### ISSUE THREAD

**Issue by freakboy3742** [pybee/voc#123]
Via Kesha Shah:
The CI server runs under Python 3.4.2, and is (usually) in a fully passing state. 

However, if you run the test suite under Python 3.4.4, you will get 3 test failures:
```
======================================================================
FAIL: test_list (tests.builtins.test_int.BuiltinIntFunctionTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/keshashah/VOC/voc-dev/voc/tests/utils.py", line 641, in func
    self.assertBuiltinFunction(x=example, f=function, operation=operation, format=self.format)
  File "/Users/keshashah/VOC/voc-dev/voc/tests/utils.py", line 661, in assertBuiltinFunction
    """ % kwargs, "Error running %(operation)s with f=%(f)s, x=%(x)s" % kwargs)
  File "/Users/keshashah/VOC/voc-dev/voc/tests/utils.py", line 373, in assertCodeExecution
    self.assertEqual(java_out, py_out, context)
AssertionError: "### [52 chars]tring or a number, not 'list'\n    test.py:3\n" != "### [52 chars]tring, a bytes-like object or a number, not 'l[17 chars]:3\n"
  ### EXCEPTION ###
- TypeError: int() argument must be a string or a number, not 'list'
+ TypeError: int() argument must be a string, a bytes-like object or a number, not 'list'
?                                           +++++++++++++++++++++
      test.py:3
 : Global context: Error running f(x) with f=int, x=[]
 
======================================================================
FAIL: test_none (tests.builtins.test_int.BuiltinIntFunctionTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/keshashah/VOC/voc-dev/voc/tests/utils.py", line 641, in func
    self.assertBuiltinFunction(x=example, f=function, operation=operation, format=self.format)
  File "/Users/keshashah/VOC/voc-dev/voc/tests/utils.py", line 661, in assertBuiltinFunction
    """ % kwargs, "Error running %(operation)s with f=%(f)s, x=%(x)s" % kwargs)
  File "/Users/keshashah/VOC/voc-dev/voc/tests/utils.py", line 373, in assertCodeExecution
    self.assertEqual(java_out, py_out, context)
AssertionError: "### [52 chars]tring or a number, not 'NoneType'\n    test.py:3\n" != "### [52 chars]tring, a bytes-like object or a number, not 'N[21 chars]:3\n"
  ### EXCEPTION ###
- TypeError: int() argument must be a string or a number, not 'NoneType'
+ TypeError: int() argument must be a string, a bytes-like object or a number, not 'NoneType'
?                                           +++++++++++++++++++++
      test.py:3
 : Global context: Error running f(x) with f=int, x=None
 
======================================================================
FAIL: test_tuple (tests.builtins.test_int.BuiltinIntFunctionTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/keshashah/VOC/voc-dev/voc/tests/utils.py", line 641, in func
    self.assertBuiltinFunction(x=example, f=function, operation=operation, format=self.format)
  File "/Users/keshashah/VOC/voc-dev/voc/tests/utils.py", line 661, in assertBuiltinFunction
    """ % kwargs, "Error running %(operation)s with f=%(f)s, x=%(x)s" % kwargs)
  File "/Users/keshashah/VOC/voc-dev/voc/tests/utils.py", line 373, in assertCodeExecution
    self.assertEqual(java_out, py_out, context)
AssertionError: "### [52 chars]tring or a number, not 'tuple'\n    test.py:3\n" != "### [52 chars]tring, a bytes-like object or a number, not 't[18 chars]:3\n"
  ### EXCEPTION ###
- TypeError: int() argument must be a string or a number, not 'tuple'
+ TypeError: int() argument must be a string, a bytes-like object or a number, not 'tuple'
?                                           +++++++++++++++++++++
      test.py:3
 : Global context: Error running f(x) with f=int, x=(1, 2.3456, 'another')
 
----------------------------------------------------------------------
```

It appears that Python 3.4.4 changed the error message raised by `int()` when invalid input is provided. Under 3.4.2, you get:
```
>>> int(list())
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: int() argument must be a string or a number, not ‘list'
```

However under Python 3.4.4, you get:
```
>>> int(list())
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: int() argument must be a string, bytes-like object or a number, not ‘list’
```

VOC should standardise on the 3.4.4 message format (since that's the format that will exist into the future); the `cleanse_python()` method in `tests/utils.py` should normalize the error string from 3.4.2 format to 3.4.4 format if it is observed in Python test output.

**Comment by keshashah** [comment]
Thanks @freakboy3742 for creating this issue on my behalf.

**Comment by gopar** [comment]
Is this issue still open? Looks like the PR is merged

**Comment by freakboy3742** [comment]
@gopar Well spotted - this was fixed in 499cd51
